lib/benchee/benchmark/runner.ex

```diff
@@ -33,10 +33,10 @@ defmodule Benchee.Benchmark.Runner do
                           scenario_context = %ScenarioContext{printer: printer, config: config}) do
     printer.input_information(input_name, config)
     printer.benchmarking(job_name, config)
-    pmap 1..config.parallel, fn ->
+    Parallel.map(1..config.parallel, fn(_) ->
```

Comment:
Ok, I've extracted the parallel mapping into a shared function. In general I'm cool with it. The only thing I don't like is having to add this unused argument here. I worry it will be confusing later on. Maybe a comment will be good enough to explain?